ci/jenkins/test.sh

```diff
@@ -163,11 +163,18 @@ function export_govc_env_var {
 
 function clean_antrea {
     echo "====== Cleanup Antrea Installation ======"
-    clean_up_one_ns "monitoring"
-    clean_up_one_ns "antrea-ipam-test-11"
-    clean_up_one_ns "antrea-ipam-test-12"
-    clean_up_one_ns "antrea-ipam-test"
-    clean_up_one_ns "antrea-test"
+    clean_ns "monitoring"
+    clean_ns "antrea-ipam-test-11"
```

Comment:
Since you are using a wildcard match, I think "antrea-ipam-test" could match all of "antrea-ipam-test-11", "antrea-ipam-test-12", "antrea-ipam-test" ?

Reply:
Yes we can only keep "antrea-ipam-test" here, updated.